lang/LangSource/GC.h

```diff
@@ -82,51 +84,47 @@ class PyrGC
 
 	MALLOC PyrObject* NewFinalizer(ObjFuncPtr finalizeFunc, PyrObject *inObject, bool inCollect);
 
-	bool IsBlack(PyrObjectHdr* inObj)  { return inObj->gc_color == mBlackColor; }
-	bool IsWhite(PyrObjectHdr* inObj)  { return inObj->gc_color == mWhiteColor; }
-	bool IsGrey(PyrObjectHdr* inObj)   { return inObj->gc_color == mGreyColor; }
-	static bool IsMarker(PyrObjectHdr* inObj) { return inObj->gc_color == obj_gcmarker; }
-	bool IsFree(PyrObjectHdr* inObj)   { return (!(IsMarker(inObj) ||
+	BOOST_FORCEINLINE bool IsBlack(PyrObjectHdr* inObj)  { return inObj->gc_color == mBlackColor; }
```

Comment:
If I'm [reading the docs right](https://gcc.gnu.org/onlinedocs/gcc-4.0.4/gcc/Function-Attributes.html), this will inline these functions even in debug builds. Where these functions not being inlined by the compiler before? Is there a reason to specifically inline these in debug? Otherwise, ```inline``` should be enough.